state/atoms/measureBundle.ts

```diff
@@ -3,7 +3,10 @@ import { atom } from 'recoil';
 /**
  * Atom tracking and controlling the value of uploaded measure bundle
  */
-export const measureBundleState = atom<File | null>({
+export const measureBundleState = atom<{ name: string; content: fhir4.Bundle | null }>({
```

Comment:
Could move the `{ name: string; content: fhir4.Bundle }` to it's own interface to neaten this up a bit.